typedoc-theme/assets/css/hifi.css

```diff
@@ -34,3 +39,37 @@
 body {
     font-family: 'Graphik', "Segoe UI", sans-serif;
 }
+
+@media screen and (min-width: 600px) {
```

Comment:
why is this CSS rule necessary? it results in the search bar being quite small on many desktop screens, which looks strange. if you didn't have this rule, what would break?

![image](https://user-images.githubusercontent.com/3409031/111669390-21437500-87ed-11eb-92c0-561427b7e0ce.png)

Reply:
this is what the same screen looks like without this rule:

![image](https://user-images.githubusercontent.com/3409031/111669454-2f919100-87ed-11eb-9706-409c00d29fb3.png)

Reply:
We wanted the search bar to be smaller. This was intentional from our discussion the other day.